src/hl7v2/schema.clj

```diff
@@ -49,6 +50,27 @@
          (drop (if (node-attrs node) 2 1))
          (into []))))
 
+(defn node-annotation [node]
+  (letfn [(rm-diacritics [s]
+            (when s
+              (-> s
+                  (Normalizer/normalize Normalizer$Form/NFD)
```

Comment:
Según entendí arriba, importaste algo de java, y en ese algo estaba `Normalizer/normalize Normalizer$Form/NFD`. Esto qué hace?

Reply:
As far as I know, this separates the characters of the encoded text, so you can get rid of diacritics if you want, which is exactly what I am doing. I mean, imagine this word `compañía`, the NDF form would be `compa~n´ia`, now you are able to remove `~` and `´` from the word and get `compania`, more accurate for comparing texts.